portal-api/api-tests/fixtures/gef/mandatoryCriteriaVersioned.js

```diff
@@ -83,20 +83,74 @@ const MANDATORY_CRITERIA_VERSIONED = [
     ],
   },
   {
-    version: 0.1,
-    createdAt: new Date('2021-01-04T00:00'),
-    updatedAt: null,
-    isInDraft: false,
-    title: 'test 4 (old version)',
-    criteria: [{ id: '1', body: 'Test is a mock test' }],
-  },
-  {
-    version: 3.4,
```

Comment:
Should we keep version 0.1 and 3.4 in here? Other versions remain

Reply:
They are not used anymore however if any tests fails I will update the tests with the latest MC version set.

Reply:
Makes sense especially as they are literally mocks with no real versioning data